src/System.Windows.Extensions/tests/Helpers.cs

```diff
@@ -0,0 +1,13 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Media.Test
+{
+    public static class Helpers
+    {
+        public const string IsSoundPlaySupported = nameof(Helpers) + "." + nameof(GetIsSoundPlaySupported);
```

Comment:
The reason why in System.Drawing we have the helpers file is because sometimes we need to use 2 conditions and ConditionalFact only supports multiple properties defined in the same class. In these case you don't need helpers.